docusaurus.config.js

```diff
@@ -81,6 +81,10 @@ module.exports = {
                 label: 'Stacks Blockchain API',
                 to: '/api',
               },
+              {
+                label: 'Token Metadata Service',
+                to: 'https://token-metadata-service-dlkidjgff-blockstack.vercel.app/',
```

Comment:
```suggestion
                href: 'https://token-metadata-service-dlkidjgff-blockstack.vercel.app/',
```

Should be an `href` here, as this is an external link

Reply:
done